src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -175,11 +192,45 @@ export class DetailsViewCommandBar extends React.Component<
         });
     }
 
-    private renderStartOverComponent(): JSX.Element {
+    private showStartOverDialog = (dialogState: StartOverDialogType) => {
+        this.setState({ startOverDialogState: dialogState });
+    };
+
+    private dismissStartOverDialog = () => {
+        this.setState({ startOverDialogState: dialogClosedState });
+    };
+
+    private startOverDialogClosed(state: DetailsViewCommandBarState): boolean {
+        return state.startOverDialogState === 'none';
+    }
+
+    public componentDidUpdate(prevProps, prevState): void {
+        // Setting focus after closing the Report Export dialog is handled in the
```

Comment:
Were the comments here just for development or will they stay in once merged to master?

Reply:
I meant for them to stay in master so we know why focus is handled differently for the start over dialog vs the export test dialog